.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,9 +4,7 @@ about: Create a report to help us improve.
 title: Bug Report
 labels: bug
 assignees: ''
-
 ---
```

Comment:
Do not include this file in the PR

Reply:
I have ignored `.github` directory overall, it was from my last commit that added extra new line at end. I'll remove it.